modules/react/dialog/stories/Dialog.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import {SymbolDoc, Specifications} from '@workday/canvas-kit-docs';
+
+import {Dialog} from '@workday/canvas-kit-react/dialog';
+import {Basic} from './examples/Basic.tsx';
+import {Focus} from './examples/Focus.tsx';
+
+<Meta title="Components/Popups/Dialog" component={Dialog} />
+
+# Canvas Kit Dialog
+
+A Dialog component is a non-Modal type of Dialog that will not render the rest of the page inert
+while it is active. A Dialog should be used in situations where the task is not critical.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic
+
+Below you will find a basic example of a Dialog. There are similarities and differences between the
+[Modal](/components/popups/modal/) and Dialog components. A Dialog does not require immeditate
+attention from the user and will not render the rest of the page inert while it is active. A
+[Modal](/components/popups/modal/) Dialog requires immediate attention and blocks interaction from
+the rest of the system.
+
+A Dialog should be used in situations where the task is not critical like in the example below.
```

Comment:
I added extra content here explaining how to adapt the Modal examples to Dialog since we're choosing not to carry examples over from the Modal docs. I also removed a bit of redundant text.

```suggestion
[Modal](/components/popups/modal/) and Dialog are very similar: most of the examples from Modal can
be adapted to Dialog by changing `Modal` to `Dialog` and replacing `Modal.Overlay` with
`Dialog.Popper`.

Unlike Modal, Dialog does _not_ render the rest of the page inert while it is active. Dialog should
be used in situations where the task does not require immediate attention such as in the example
below.
```